### PR TITLE
docs: Added LandscapeConfig keys

### PR DESCRIPTION
**Pull request by YanisaHS**
I've added some missing info. Feel free to accept/reject as you see fit, but the port number on the URL should be accepted. AFAIK, adding the port number is expected to be temporary. I'll keep your docs updated if/when that changes.

I didn't want to delete the information you had already regarding the Agent keys, but I was a bit confused by that section and what users should do with the other keys. If you want to go over it some time, feel free to MM me or schedule a meeting. I know your docs are under construction though so I think it's fine for now too :smiley: 

Also, here's the link to the Landscape Docs for registering a WSL host, if that's helpful: https://ubuntu.com/landscape/docs/register-wsl-hosts-to-landscape

### CODE REVIEW

**Comment by YanisaHS** [comment]
I looked at the failed check and it doesn't like how I spelled "Quickstart" - we call it that though (one word) since it's part of the name of the `landscape-server-quickstart` package. Hope you can ignore it :upside_down_face:

File: docs/reference/landscape.md [hunk 1]
```diff
@@ -42,13 +42,21 @@ This section is the same one you'd see in other Landscape platforms, with some c
...
 All other data is ignored by the Agent and passed on verbatim to the Landscape client. You can see the reference for all the other keys in the [Landscape repository](https://github.com/canonical/landscape-client/blob/master/example.conf).
 
+The following keys are used by the Landscape client:
+
+- `account_name`: The Landscape account name this computer belongs to. For self-hosted Landscape accounts, the account name defaults to “standalone”.
+- `url`: The main URL for the Landscape Server to connect this client to. This defaults to the URL of your Landscape account followed by `/message-system`.
+- `ping_url`: The ping URL you want this client to report to. This defaults to the URL of your Landscape account followed by `/ping`.
+- `log_level`: This can be one of: "debug", "info", "warning", "error", "critical"
+- (Optional) `registration_key`: An optional account-wide key used to register new clients. There is no key defined by default, but one can be set in your Landscape account.
+
```
Comment (EduardGomezEscandell) [inline docs/reference/landscape.md:52]: I'm not sure we want to keep this duplicated info here. Our idea was to do like a layered documentation:
- Link to Landscape's docs for stuff that does not change
- Document what we add/change

You're the expert though; what do you think is best?

File: docs/reference/landscape.md [hunk 2]
```diff
@@ -42,13 +42,21 @@ This section is the same one you'd see in other Landscape platforms, with some c
...
 ### Example
 
 Here is an example of a self-hosted Landscape configuration:
 
 ```ini
 [host]
-url = https://landscape-server.domain.com/hostagent
+url = https://landscape-server.domain.com/6554
```
Comment (EduardGomezEscandell) [inline docs/reference/landscape.md:59]: This does not match the previous explanantion:
> The URL of your Landscape account followed by a colon (`:`) and the port number

**Review by EduardGomezEscandell — CHANGES_REQUESTED** [review]
Thanks for this :D

Regarding the spellcheck: you can add it to `./docs/.custom_wordlist.txt` and the CI will pass.

File: docs/reference/landscape.md (hunk 1 above)
Comment (tmihoc) [reply docs/reference/landscape.md:52]: I wouldn't duplicate. Instead, I'd link to the upstream and then document just the diff (clarifying that it's just the diff).

**Comment by jibel** [comment]
> I looked at the failed check and it doesn't like how I spelled "Quickstart" - we call it that though (one word) since it's part of the name of the `landscape-server-quickstart` package. Hope you can ignore it 🙃

You can add Quickstart to the wordlist docs/.custom_wordlist.txt

File: docs/reference/landscape.md (hunk 1 above)
Comment (YanisaHS) [reply docs/reference/landscape.md:52]: Thanks both! @tmihoc I like that set up - I'll link it to Landscape docs instead

**Comment by YanisaHS** [comment]
@EduardGomezEscandell I've updated the doc, but my link check didn't pass, although the link I added does work. I think it might be because I linked to a heading? Is this a bug in the automated check, or do you not want links going directly to headings? (I think it makes more sense to go to the specific section for this one)

**Comment by jibel** [comment]
> @EduardGomezEscandell I've updated the doc, but my link check didn't pass, although the link I added does work. I think it might be because I linked to a heading? Is this a bug in the automated check, or do you not want links going directly to headings?
> 
> I think it makes more sense to go to the specific section for this one, but I can also reword it and link to the main page if that's preferred.

I think it was bad luck and a one-off failure. I re-ran the linkchecker and the tests passed. I also tested locally with ```make linkcheck``` and it didn't fine any issue.

File: docs/reference/landscape.md [hunk 3]
```diff
@@ -57,6 +57,8 @@ account_name = standalone
 log_level = debug
 ```
 
+For more information, see [the Landscape guide on how to configure Ubuntu Pro for WSL](https://ubuntu.com/landscape/docs/register-wsl-hosts-to-landscape#heading--configure-ubuntu-pro-for-wsl-for-landscape).
```
Comment (EduardGomezEscandell) [inline docs/reference/landscape.md:60]:
```suggestion
For more information, see [the Landscape guide on how to configure Ubuntu Pro for WSL](https://ubuntu.com/landscape/docs/register-wsl-hosts-to-landscape/#heading--configure-ubuntu-pro-for-wsl-for-landscape).
```

Linkcheck did fail for me as well. Seems like adding a `/` before the anchor solves the issue.

**Review by EduardGomezEscandell — CHANGES_REQUESTED** [review]
Almost there

**Pull request merged**